sdks/jreleaser-bitbucket-cloud-java-sdk/gradle.properties

```diff
@@ -0,0 +1,20 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2020-2023 The JReleaser authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+project_description = Java SDK for Bitbucket Cloud
+automatic.module.name = org.jreleaser.sdk.bitbucketCloud
```

Comment:
I think there's a preference to use lowercase names

Reply:
FWIW module names with hyphens are not allowed. It'd be better to name the module as `org.jreleaser.sdk.bitbucketcloud`.